perf/lua/1mops_write.lua

```diff
@@ -0,0 +1,325 @@
+#!/bin/env tarantool
+
+local err, popen = pcall(require,'popen')
+if err ~= true then
+    print(err)
+    popen = {}
+end
+local clock = require('clock')
+local fiber = require('fiber')
+local math = require('math')
+local json = require('json')
+local fio = require('fio')
+
+local params = require('internal.argparse').parse(arg, {
+    {'engine', 'string'},
+    {'fibers', 'number'},
+    {'index', 'string'},
+    {'nodes', 'number'},
+    {'nohint', 'boolean'},
+    {'ops', 'number'},
+    {'sync', 'boolean'},
+    {'transaction', 'number'},
+    {'warmup', 'number'},
+    {'help', 'boolean'},
+    {'h', 'boolean'},
+})
+
+
+if params.help or params.h then
+    print([[
+
+ Usage: taskset 0xef tarantool 1mops_write.lua [options]
+
+ Options can be used with '--', followed by the value if it's not
+ a boolean option. The options list with default values:
+
+   engine <string, 'memtx'>  - select engine type
+   fibers <number, 50>       - number of fibers to run simultaneously
+   help (same as -h)         - print this message
+   index <string, 'HASH'>    - select index type
+   nodes <number, 1>         - number of nodes to test one-way replication
+   nohint <boolean>          - to turn the index hints off
+   ops <number, 10000000>    - total amount of replaces to be performed
+   sync <boolean>            - to turn the synchro replication on
+   transaction <number, 150> - number of replaces in one transaction
+   warmup <number, 10>       - percent of ops to skip before measurement
+
+ Being run without options tests the HASH index of memtx engine by
+ running of 50 fibers that replace 150 integers using each as a primary
+ key. Refer to the 'fiber_load' function below to change the workload.
+]])
+    os.exit(0)
```

Comment:
Nit: I'd rather use `exit` helper here either, however, this is totally minor, so feel free to ignore.

Reply:
Done.